### ISSUE TITLE
How to add context in this format ? is it possible.

### ISSUE THREAD

**Issue by Muthurajj** [danubetech/verifiable-credentials-java#26]
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    **{
      "dk": "https://ld.dock.io/credentials#",
      "BasicCredential": "dk:BasicCredential",
      "name": "dk:name",
      "title": "dk:title",
      "description": "dk:description",
      "logo": "dk:logo"
    }**
  ],

**Comment by peacekeeper** [comment]
No, right now it only supports context URIs, not inline context objects, see here: https://github.com/decentralized-identity/jsonld-common-java/blob/4d296845d9a2af268b179b6f995fcc2bbad04454/src/main/java/foundation/identity/jsonld/JsonLDObject.java#L70
